src/shared/organisms/DataExplorerGraphFlowContent/DataExplorerGraphFlowContent.tsx

```diff
@@ -13,7 +13,7 @@ const DataExplorerContentPage = ({}) => {
   return (
     <div className="degf-content__wrapper">
       {fullscreen ? (
-        <Fragment>
+        <>
```

Comment:
for `Fragment` vs `<></>`, it's a personal preference 
one benefit of Fragment is that you can use keys  when you need it as mapping a list

Reply:
It's true. If no mapping is needed, then I think the shorthand looks cleaner as it implies that there won't be a DOM node added in the rendered HTML.

Reply:
That's what I said so basically if someone use the fragment we do not need to rename it (ex: For me fragment is more clean ) it's clear and understandable for all the devs that fragment === <></>